doc/cephadm/install.rst

```diff
@@ -45,7 +46,7 @@ curl-based installation
   
   .. prompt:: bash #
 
-   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/cephadm/cephadm
```

Comment:
```suggestion
   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/|stable-release|/src/cephadm/cephadm
```

see https://github.com/ceph/ceph/blob/master/doc/install/get-packages.rst#debian-packages for an example, and the rendered document looks like https://docs.ceph.com/en/latest/install/get-packages/ .

because we always substitute `|stable-release|` with the value read from `doc/releases/releases.yml`.

see https://github.com/ceph/ceph/blob/f36dd3f5f2e07dc8358496f849af180e6dbd3622/doc/conf.py#L139-L142

and 
https://github.com/ceph/ceph/blob/f36dd3f5f2e07dc8358496f849af180e6dbd3622/doc/conf.py#L24-L28

Reply:
Thanks, good idea. Fixed with latest version. I also fixed a couple of other places where we had hardcoded "octopus".